src/formatter.cpp

```diff
@@ -20,6 +20,47 @@
 //' @return A single (formatted) string
 //' @seealso https://github.com/fmtlib/fmt
 // [[Rcpp::export]]
```

Comment:
I also noticed that now:  you are breaking an Rcpp convention here.  The `// [[Rcpp::export]]` must directly precede the function signature so you may have to move the #ifdef parts inside of `std::string formatter(const std::string s, std::vector<std::string> v) {`.